component-definition.json

```diff
@@ -1,257 +1,312 @@
 {
-  "groups": [
```

Comment:
@Samriddhi-1998 This file is totally changed and its hard to track your changes, please rollback and add your changes only.